platform/ios/app/MBXSnapshotsViewController.m

```diff
@@ -50,12 +57,13 @@ - (MGLMapSnapshotter*) startSnapshotterForImageView:(UIImageView*) imageView coo
     options.zoomLevel = 10;
     
     // Create and start the snapshotter
+    __weak UIImageView *weakImageView = imageView;
```

Comment:
There was a leak here because we were retaining `self` when assigning the image to the imageView.